Xamarin.Forms.Platform.Tizen/Renderers/EditorRenderer.cs

```diff
@@ -92,5 +93,13 @@ void UpdateKeyboard(bool initialize)
 
 			Control.Keyboard = Element.Keyboard.ToNative();
 		}
+
+		string MaxLengthFilter(Entry entry, string s)
+		{
+			if (s.Length <= Element.MaxLength)
```

Comment:
Please change the condition as below.
```
if (entry.Text.Length < Element.MaxLength)
```
This is because the lengh of `s` is always 1.

Reply:
Thanks! Do I still return `s`? I figure `s` is then the character that is typed? Just out of curiosity: what does `PrependMarkUpFilter` or `AppendMarkUpFilter` do? One is triggered before hand and one after? Or does it say anything about in which order filters are applied?

Reply:
Yes, `s` is the character typed by user.  This filter is called each time a text is inserted in the entry. So, you should return `s` if it less than the length of entry's text. 

The difference of `PrependMarkUpFilter` and 'AppendMarkUpFilter' is that the order in which the filters are added in different. Suppose that there are 3 filters - `FilterA`, `FilterB`, and `Filter C`- exists. In this situaltion, if you try to apply a new `FilterD` with `PrependMarkUpFilter`, then the order of filtes is `FilterD - FilterA -FilterB - FilterC`.  Otherwise, with `AppendMarkUpFilter`, `FilterA - FilterB- FilterC - FilterD`.

Reply:
Great! Thanks!